clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml

```diff
@@ -0,0 +1,66 @@
+kind: List
+apiVersion: v1
+items:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: ipi-deprovision
+    namespace: ci
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: ipi-deprovision
+    namespace: ci
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ipi-deprovision:latest
```

Comment:
Is it conflicting to the following?

https://github.com/openshift/release/blob/9d75912f1577b2a4d97195a22b9e565a13c6e47c/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L207-L218

Reply:
I have another PR opened to remove the build in ci-operator config

Reply:
https://github.com/openshift/release/pull/17069